### DISCUSSION TITLE
How to connect Postal With Mautic

### DISCUSSION THREAD

**Discussion by malikhamza-v** [postalserver/postal#2589] [Help with using Postal]
Hi, 

I want to use mautic for sending emails with my Postal SMTP Server. But I'm unable to do so. I am trying to connect with these settings.

![image](https://github.com/postalserver/postal/assets/106825125/6b7997be-83de-4fa2-8e07-6534d61a8bcb)

I am getting this error on Mautic

![image](https://github.com/postalserver/postal/assets/106825125/a4a00757-123d-4bff-9269-c45f5cf4cd89)

My Postal is hosted on Contabo. And my Mautic is hosted on a Shared Hosting. Now I'm not sure but I think the problem is Port 25 is open on Contabo but not on my Shared Hosting. Is there any solution for it? 

Edit: I tried connecting on my local machine with the above credentials but I am also unable to do so.

**Comment by willpower232** [comment]
You can confirm with telnet from a different computer whether or not your port 25 is open to the internet. It is possible that either you need to open it in a firewall or you need to ask your server provider to open the port for you.

Alternatively, you could try changing the "SMTP encryption type" to be "off" or whatever the equivalent is as Postal only supports explicitly upgrading to SSL connections rather than implicitly having them by default.
